client/fns.go

```diff
@@ -252,3 +252,69 @@ func (c *Cmd) Outputs() ([]bytes.Buffer, error) {
 	}
 	return r[:], nil
 }
+
+// parseBinds parses a CPU_NAMESPACE-style string to a
+// an fstab format string. It is not intended to be called
+// outside this package, because it is sensitive to values
+// in the Cmd struct (e.g. c.TmpMnt) and users might set things
```

Comment:
is this bit about c.TmpMnt still true?  since you're passing in TmpMnt as a string, and not as 'c' directly.  regardless, keeping it in this package makes sense to me.

Reply:
I thought about changing this, but I decided to leave it, as it kind of points back to the reason.